pkg/vadmin/restart_node_vc.go

```diff
@@ -30,9 +30,9 @@ import (
 // lost cluster quorum. The IP given for each vnode may not match the current IP
 // in the vertica catalogs.
 func (v *VClusterOps) RestartNode(ctx context.Context, opts ...restartnode.Option) (ctrl.Result, error) {
```

Comment:
I kept this because this function implements an interface that is also used by the Admintools operator.

Reply:
The question is @spilchen now that we are moving away from `admintools` should we replace `restartnode*` with `startnode*`

Reply:
Possibly. But I think we can fix that up in the future. There are already cases where the operator dispatcher interface name doesn't match the vclusterops API name.